core/src/main/java/com/mygdx/game/Managers/SaveManager.java

```diff
@@ -0,0 +1,90 @@
+package com.mygdx.game.Managers;
+
+import com.badlogic.gdx.Gdx;
+import com.badlogic.gdx.Preferences;
+import com.mygdx.game.Components.Pirate;
+import com.mygdx.game.Components.Transform;
+import com.mygdx.game.Entitys.College;
+import com.mygdx.game.Entitys.Ship;
+
+import java.util.ArrayList;
+
+/**
+ * Added for assessment 2
```

Comment:
Since we added the whole class here, do we have to comment "Added for Assessment 2" on each method?